HHS/Head-Start-TTADP
# [TTAHUB-3262] Update AR page state on save draft (only if "not started")

## Description of change
Update the Activity Report page state on "save draft" for goals

## How to test
Create a report, fill out recipients on first page, add a goal, save draft (not save and continue).
Return to landing page, clear local storage, return to report.
Goals and objectives should show "in progress"

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-3262


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
